lib/components/position_component.dart

```diff
@@ -212,4 +212,6 @@ abstract class PositionComponent extends Component {
   void clearChildren() {
     _children.clear();
   }
+
+  OrderedSet<Component> get getChildren => _children;
```

Comment:
I would:
1. call it `children`
2. copy

now, do you have an example of what you are trying to do? I wonder if there is something else we can expose to make it even easier (so the children knows what they need to know)

Reply:
example added to PR comments